app/src/main/res/layout/hero_item.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content">
+
+    <ImageView
+        android:id="@+id/hero_image"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:contentDescription="@string/hero_image"
+        android:src="@mipmap/ic_launcher"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <TextView
+        android:id="@+id/hero_text"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="@dimen/space_between_items"
+        android:background="@color/primaryColor"
+        android:ellipsize="end"
+        android:gravity="center"
+        android:maxLines="2"
```

Comment:
poner un minimo de 2 lineas tambien en caso de que uno tenga nombre de una linea y el de al lado tiene nombre de 2 lineas apareceria un hueco innecesario.

Reply:
Hey @JorgeGreek you are referring to `minLines` because `maxLines` just defines a maximum number of lines